packages/back-end/src/api/openapi/paths/postAttribute.yaml

```diff
@@ -0,0 +1,54 @@
+tags:
+  - attributes
+summary: Create a new attribute
+operationId: postAttribute
+x-codeSamples:
+  - lang: "cURL"
+    source: |
+      curl -X POST https://api.growthbook.io/api/v1/attributes \
+        -d '{"property": "foo", "datatype": "boolean", "description": "My new attribute" }' \
+        -u secret_abc123DEF456:
+requestBody:
+  required: true
+  content:
+    application/json:
+      schema:
+        type: object
+        required:
+          - property
+          - datatype
+        properties:
+          property:
+            type: string
+            description: The attribute property
+          datatype:
+            type: string
+            enum: ["boolean", "string", "number", "secureString", "enum", "string[]", "number[]", "secureString[]"]
```

Comment:
I haven't worked with our yaml API schemas much, is there a way to make these enums reference a definition elsewhere? (Or vice-versa, change the other definition to reference this instead of duplicating it?)

Reply:
I don't think so. But the typechecker will fail if this gets out of sync, I've had it happen with `isoCountryCode` below.